data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -160,3 +160,51 @@
 EJSCREEN_AREAS_OF_CONCERN_STATE_95TH_PERCENTILE_COMMUNITIES_FIELD_NAME = (
     "EJSCREEN Areas of Concern, State, 95th percentile (communities)"
 )
+
+# Community Threshold Field Names for Derived columns (Issue 921)
+
+# Climate Change
+FEMA_LOSS_RATE_LOW_INCOME = "At or above the 90th percentile for FEMA's expected loss rate and is low income"
```

Comment:
Can we programatically create these using f-strings, so they update automatically if we change the threshold? 

That'll also help with consistency (e.g., the PM2.5 field has a different name style than every other field).

Reply:
@lucasmbrown-usds - Right, as I mentioned in the description, I am not happy either about the amount of boilerplate. 
I have some ideas to reduce the amount of boilerplate but wanted to just ensure this meets the basic requirements we discussed this morning.

Reply:
Any reason not to do this now?